<launch>
	<node pkg="random_movement" type="random_movement_node" name="random_movement_node" output="screen">	
	  	<param name="angle_front"		type="int" 		value="35" />
	  	<param name="angle_back"		type="int" 		value="35"/>
	  	<param name="steering_angle"		type="int" 		value="90" />		
	  	<param name="break_distance"		type="double"		 	value="0.6"/>
	</node>
</launch>
